app/models/infection.rb

```diff
@@ -0,0 +1,4 @@
+class Infection < ApplicationRecord
+  belongs_to :user
+  belongs_to :organism
```

Comment:
why is this not :through?

Reply:
you have access to it through user:id and organism:id of the infections